### PR TITLE
PRM Automated Test - LPS-195725 | master

### PR DESCRIPTION
**Pull request by brunadmelo**
[LPS-195725](https://liferay.atlassian.net/browse/LPS-195725)

### CODE REVIEW

**Comment by brunadmelo** [comment]
[LPS-195725](https://liferay.atlassian.net/browse/LPS-195725)

**Comment by brunadmelo** [comment]
ci:test:sf

**Comment by brunadmelo** [comment]
cc: @kenduenwald, @renato-araujo-liferay, @guivilarim

**Comment by guivilarim** [comment]
ci:test:osb-site-initializer-partner-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase [hunk 1]
```diff
@@ -547,4 +547,41 @@ definition {
...
+	test PMUUserCanDeleteMDFClaimInDraftStatus {
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given that the PMU user goes to the created MDF") {
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+
+			WaitForElementPresent(
+				key_campaignName = "Campaign Name",
+				locator1 = "PRMMDFRequest#REQUEST_ID");
```
Comment (guivilarim) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase]: Hey @brunadmelo , is this `WaitForElementPresent` really necessary? If you open the MDF Requests Site Page the  Campaign Name should already be there.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro [hunk 2]
```diff
@@ -115,6 +115,20 @@ definition {
 			locator1 = "Header#H1_TITLE");
 	}
 
+	macro deleteClaim {
+		Button.click(button = "Delete");
+
+		Confirm(value1 = "Are you sure?");
+
+		PRMNavigator.openSitePage(pageName = "MDF Claims");
+
```
Comment (guivilarim) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro:124]: Also @brunadmelo , could you separate the first two actions on the macro to make more generic and possible to delete a claim also with this? By separating the Button.click and Confirm is it possible to make more generic.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase (hunk 1 above)
Comment (brunadmelo) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase]: @guivilarim indeed this is not necessary, I will remove this assert. Was using that to run the test on my home connection which is more slow than the one from the Liferay office.

**Comment by brunadmelo** [comment]
ci:test:sf

**Comment by brunadmelo** [comment]
ci:test:sf

**Comment by guivilarim** [comment]
ci:forward

**Pull request closed without merging**